docs/content/commands/rclone_dedupe.md

```diff
@@ -13,8 +13,8 @@ Interactively find duplicate files and delete/rename them.
 
 
 By default `dedupe` interactively finds duplicate files and offers to
-delete all but one or rename them to be different. Only useful with
-Google Drive which can have duplicate file names.
+delete all but one or rename them to be different.  Only some backends
+(e.g. Google Drive, Opendrive) have the ability to store duplicate file names.
```

Comment:
I'm not sure the Opendrive isn't a bug?

Google Drive and Mega definitely can and have rclone dedupe support. Opendrive doesn't have dedupe support yet so can't actually run dedupe.

Reply:
I don't know for sure if Opendrive is a bug or unsupported feature, but dupes and `rclone dedupe` (of files at least) seems to work.

Reply:
I removed "Opendrive" from this line

Reply:
@eharris I propose to keep Google Drive and keep questionable Opendrive for future. It really isn't importand at all.

Reply:
This small change hangs for too long. Let's have it in.

Reply:
Dupes are still a problem on Opendrive, and the `rclone dedupe` command does currently work (as I made clear in the inline thread) so I'm unclear as to why you want to remove Opendrive from being mentioned as a backend that has this issue.  What exactly are the changes you want me to make?